api/collections/serializers.py

```diff
@@ -0,0 +1,113 @@
+from rest_framework import serializers as ser
+from api.base.serializers import JSONAPISerializer, CollectionLinksField, Link, LinksField
+from website.models import Node
+from framework.auth.core import Auth
+
+class CollectionSerializer(JSONAPISerializer):
+    filterable_fields = frozenset(['title'])
+
+    id = ser.CharField(read_only=True, source='_id')
+    title = ser.CharField(required=True)
+    date_created = ser.DateTimeField(read_only=True)
+    date_modified = ser.DateTimeField(read_only=True)
+    modified_by = ser.CharField(read_only=True, source='get_modified_by')
+    links = CollectionLinksField({
+        'children': {
+            'related': Link('collections:collection-children', kwargs={'pk': '<pk>'}),
```

Comment:
This is happening elsewhere for existing routes, but go ahead and change `pk` to `collection_id`. This will happen in your urls file as well (and I'll likely comment on that when I get there). This will make documentation better.